spec/models/observation_collection_spec.rb

```diff
@@ -1,9 +1,43 @@
 require 'spec_helper'
 
 describe ObservationCollection do
-  describe ".new" do
-    xit "" do
-      # add tests
+  describe ".new_from_array" do
+    it "returns new ObservationCollection" do
```

Comment:
Good points> I should've tested to see if it `responds_to(:each)` instead of being an Enumerable. I'll also rename the test and break it out into two.